test/mlir/accelerators/nnpa/module_op_be/lit.local.cfg

```diff
@@ -0,0 +1,7 @@
+root = config.root
+# enable when onnx-mlir driver works well with nnpa.
+config.unsupported = True
+# if sys.byteorder == "little":
```

Comment:
Same here and in the other cfg files - it's better not to include commented out code